x-pack/plugins/security_solution/scripts/endpoint/resolver_generator_script.ts

```diff
@@ -4,14 +4,40 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 /* eslint-disable no-console */
-import * as path from 'path';
 import yargs from 'yargs';
-import * as url from 'url';
-import fetch from 'node-fetch';
 import { Client, ClientOptions } from '@elastic/elasticsearch';
 import { ResponseError } from '@elastic/elasticsearch/lib/errors';
+import { KbnClient, ToolingLog } from '@kbn/dev-utils';
+import { AxiosResponse } from 'axios';
+import { KibanaConfig } from '@kbn/dev-utils/target/kbn_client/kbn_client_requester';
```

Comment:
Can you check if this can be imported directly from `@kbn/dev-utils`? if so, then just add it to the import above.

Reply:
i don't think it can be